mllib/src/main/scala/org/apache/spark/mllib/api/python/PythonMLLibAPI.scala

```diff
@@ -763,11 +764,13 @@ private[python] class PythonMLLibAPI extends Serializable {
       maxBins = maxBins,
       categoricalFeaturesInfo = categoricalFeaturesInfo.asScala.toMap)
     val cached = data.rdd.persist(StorageLevel.MEMORY_AND_DISK)
+    // Only done because methods below want an int, not an optional Long
+    val intSeed = getSeedOrDefault(seed).toInt
```

Comment:
yeah, I was trying to say that converting to a smaller number of bits would cause some be more likely than others, but I agree in practice it's not going to make any difference here.